modules/uhttp/doc.go

```diff
@@ -30,7 +30,6 @@
 //     "io"
```

Comment:
Did you mean to commit this? You may need to rebase, I think @madhuravi just changed this